tests/__init__.py

```diff
@@ -29,8 +30,12 @@
 }
 
 
-def get_bundle_fqid():
-    return f"{uuid.uuid4()}.{get_version()}"
+def get_bundle_fqid() -> BundleFQID:
+    return BundleFQID(uuid=uuid.uuid4(), version=get_version())
```

Comment:
This looks odd. Don't you wwant `str(uuid.uuid4())` ?

Reply:
✅